src/main/scala/com/databricks/labs/overwatch/api/ApiCallV2.scala

```diff
@@ -539,6 +543,37 @@ class ApiCallV2(apiEnv: ApiEnv) extends SparkSessionWrapper {
     }
   }
 
+  def asRawDF() : DataFrame = {
```

Comment:
@sriram251-code  Can we move this function to PipelineTable.scala like we have for asDF()